apps/comps/app/competitions/[id]/page.tsx

```diff
@@ -94,8 +94,11 @@ export default function CompetitionPage({
   }) => (
     <Button
       disabled={competition.status !== "pending" || disabled}
-      variant="ghost"
-      className={className}
+      variant="default"
+      className={cn(
+        "border border-blue-500 bg-blue-500 text-white hover:bg-white hover:text-blue-500",
```

Comment:
```suggestion
        "border border-blue-500 bg-blue-500 text-white hover:bg-white hover:text-blue-500 disabled:hover:bg-blue-500 disabled:hover:text-white",
```
when disabled, i think we want the btn to _not_ change bg/text colors on hover

https://github.com/user-attachments/assets/dbab2b60-c26f-40e9-903f-d366272a276a